test/testlib.sh

```diff
@@ -252,13 +252,13 @@ setup_test_data () {
 
   mkdir -p "$loc/audit-log/"
   cd "$loc/audit-log/"
-  touch audit_log-1-$last_yr-$last_mth-1.gz
-  touch audit_log-1-$this_yr-$this_mth-1.gz
+  echo "fake audit log last yr last mth" | gzip > audit_log-1-$last_yr-$last_mth-1.gz
+  echo "fake audit log this yr this mth" | gzip > audit_log-1-$this_yr-$this_mth-1.gz
 
   # Create hookshot logs
   mkdir -p "$loc/hookshot/"
   cd "$loc/hookshot/"
-  touch hookshot-logs-2018-03-05.gz
+  echo "fake hookshot log" | gzip > hookshot-logs-2018-03-05.gz
```

Comment:
Should this be a hard-coded date-based filename?

Reply:
Sort of. The filename needs to be in the format: `<index_name>-<year>-<mth>-<day>.gz` as this is how we name the file when backing up, so the actual date isn't really relevant. It's simpler to use a hard-coded date then try get fancy with dynamically generating it.  We're only interested in the fact it's being processed in the test, regardless of its filename.